components/feature/prompts/src/main/java/mozilla/components/feature/prompts/PromptFeature.kt

```diff
@@ -240,6 +264,20 @@ class PromptFeature private constructor(
                         prompt.dismiss()
                     }
                     activePrompt?.clear()
+                    loginPicker.dismissCurrentLoginSelect()
+                }
+        }
+
+        sessionPromptScope = store.flowScoped { flow ->
```

Comment:
This was added for dismissing prompts when a new tab is selected (ie via the swipe gesture or long press to open a new tab). Is this the best thing to observe on for that?